app/views/course/reference_timelines/_reference_timeline.json.jbuilder

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+json.id reference_timeline.id
+json.title reference_timeline.title
+json.weight reference_timeline.weight
+
+if render_times
```

Comment:
I dont see line 8 to line 18 being typed in `TimelineListData`?

Reply:
The attributes in lines 8–18 are (currently) used if the reference timeline was created using nested attributes for the reference times. Currently, the Timeline Designer does not use this, nor even use the `:show` action. Hence, it was not typed.

Reply:
Then we just remove it?